sentry-core/src/test/java/io/sentry/core/transport/HttpTransportTest.kt

```diff
@@ -297,6 +297,66 @@ class HttpTransportTest {
         assertTrue(transport.isRetryAfter("event"))
     }
 
+    @Test
+    fun `parse X-Sentry-Rate-Limit and ignore unknown categories`() {
+        val transport = fixture.getSUT()
+
+        whenever(fixture.connection.inputStream).thenThrow(IOException())
+        whenever(fixture.connection.getHeaderField(eq("X-Sentry-Rate-Limits")))
+            .thenReturn("60:default;wtf;error;security:organization")
+
+        val event = SentryEvent()
+
+        transport.send(event)
+        Thread.sleep(2000)
```

Comment:
`m`: I don't think we need this `Thread.sleep(2000)` here.

Reply:
good catch, copy pasta error